.ci-orchestrator/steps/fats/liteCheckpointFATs.yml

```diff
@@ -0,0 +1,33 @@
+# This step runs any FATs which declare that they test the 'checkpoint' feature in an environment with CRIU installed.
+# Checkpoint FATs therefore run twice in builds but the @CheckpointTest annotation and use of the fat.test.run.checkpoint.only property mean that in CRIU environments
+# they only run the test cases with @CheckpointTest (skipping the others) and in non-CRIU environments they only run the unannotated test cases. Hence, there's no duplication of testing.
+- stepName: Checkpoint FATs (Lite)
+  conditionalRun:
+    - type: ifTrue
+      value: ${spawn.checkpoint}
+  workType: FAT
+  timeoutInMinutes: 1920
+  properties:
+    spawn.checkpoint: true
```

Comment:
I don't know if this is important enough to warrant a change, but in other files with `spawn.checkpoint` property, its value is `"true"` (i.e., no surrounding quotation marks here).